apps/server/src/modules/tldraw/utils/callback.ts

```diff
@@ -0,0 +1,75 @@
+const http = require('http')
+const CALLBACK_URL = process.env.CALLBACK_URL ? new URL(process.env.CALLBACK_URL) : null
+const CALLBACK_TIMEOUT = process.env.CALLBACK_TIMEOUT || 5000
+const CALLBACK_OBJECTS = process.env.CALLBACK_OBJECTS ? JSON.parse(process.env.CALLBACK_OBJECTS) : {}
```

Comment:
Please use the _Configuration_ class from _@hpi-schul-cloud/commons_ for retrieving all the configuration variables (including env vars).